src/commons/application/types/SessionTypes.ts

```diff
@@ -171,6 +174,7 @@ export type CourseConfiguration = {
   sourceChapter: Chapter;
   sourceVariant: Variant;
   moduleHelpText: string;
+  defaultPrompt: string;
```

Comment:
Likewise, `defaultPrompt` is too vague. As discussed previously, let us rename this to `defaultLlmPrompt` or something.